api/message.go

```diff
@@ -45,6 +45,14 @@ type pagingParams struct {
 	Since uint `form:"since" binding:"min=0"`
```

Comment:
@nienow would something like a raw message API solve your problem?

E.g. title, extras, title can be optionally sent as header values, and the full message body is the gotify message.
```http
POST /message/raw
Content-Type: text/plain
X-Gotify-Title: my title
X-Gotify-Priority: 10
X-Gotify-Extras: { "client::display": { "contentType": "text/markdown" } }

this is the message **body**....
```

Reply:
No, home assistant cannot pass variables into headers